src/middlewared/middlewared/plugins/account_/internal_cache.py

```diff
@@ -20,6 +20,39 @@ async def get_builtin_user_id(self, username):
         except KeyError:
             raise CallError(f'{username!r} user not found', errno.ENOENT)
 
+    @private
+    async def username_to_uid(self, user_name):
```

Comment:
This basically duplicates `user.get_user_obj` We should just use the public method. It raises KeyError if user doesn't exist.